ci/fileUtil.sc

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env amm
 
+import $ivy.`javax.xml.bind:jaxb-api:2.3.1`
```

Comment:
What's the reason for this?

Reply:
looks like the build switched to java > 10? Not sure, but it started failing because this was removed in Java 11

Reply:
Ah, dang. That's probably because the build used a node from the `mesos-plugin`. Let's remove this change here and add it in a separate PR.
```suggestion
```

Reply:
but then I cannot merge 😢 let's YOLO the other change then :D